scripts/globals/mobskills/bloody_caress.lua

```diff
@@ -0,0 +1,34 @@
+-----------------------------------
+--  Bloody Caress
+--
+--  Delivers a threefold attack. Additional effect: Drain
+--  Type: Physical
+--  100% TP: ??? / 250% TP: ??? / 300% TP: ???
+--
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 3
+    local accmod = 1
+    local dmgmod = 1
+
+    -- TODO: Once `Floral Bouquet` TP move is implemented, this skill is eligible to target
```

Comment:
Ty for the todo note, and thank you for conservative mod data on initial implementation.